src/pages/CivicProfile.jsx

```diff
@@ -33,7 +33,7 @@ const CivicProfile = () => {
           </MenuList>
         </nav>
       </Container>
-      <Container>
+      <Container sx={{ width: '60dvw' }}>
```

Comment:
It really shouldn't be necessary to add styling to `CivicProfile.js`, especially when it then affects other areas that utilize it. Notice how now `BasicProfile.jsx` went from this:

<img width="239" alt="2024-05-18 (7)" src="https://github.com/codeforpdx/PASS/assets/83156697/16ae070a-28e5-4fbf-ac4a-04586abcf905">

To this:

<img width="216" alt="2024-05-18 (8)" src="https://github.com/codeforpdx/PASS/assets/83156697/6142d0ee-45b5-4fb3-b810-1f85e0b39a55">

Reply:
Removed styling on bottom container in src/pages/CivicProfile.jsx per Andy C.W. comment on PR #633.